Made changes to handle header to show when rendering a custom accordion view of the data grid

rameshkumarit1990 opened the pull request:
-- Made changes to show header when rendering a custom accordion view of the data grid
-- The title width should be aligned with the content of our custom content so pass width value from column definition.

rameshkumarit1990 committed:
Support custom images for sorting header and pagination icons.

-- Can customize the sorting header icon, pagination right left icons and pages count using css from portal

rameshkumarit1990 committed:
Merge branch 'ydsood:feature-redux_form_field' into feature-redux_form_field

rameshkumarit1990 committed:
#0000 Made changes to show header when rendering a custom accordion view of data grid

-- Made changes to show header when rendering a custom accordion view of data grid
-- The title width should be aligned to with content of our custom content so pass width value from column definition.

buptlee commented:
rather than naming it as accordionComponent, can we rename it as show table header? the prop should be generic and shouldn't tie with the client's side implementation.

also since we have a new prop to control the table header, I would rather remove the cellComponent from that if statement.

the version should be 0.2.0-alpha.5.0 as we introduce a new prop and will impact the existing client code,

buptlee reviewed (changes requested):
rather than naming it as accordionComponent, can we rename it as show table header? the prop should be generic and shouldn't tie with the client's side implementation.

also since we have a new prop to control the table header, I would rather remove the cellComponent from that if statement.

the version should be 0.2.0-alpha.5.0 as we introduce a new prop and will impact the existing client code,

rameshkumarit1990 committed:
Added the hideTableHeader prop and updated version

-- Updated version
-- Added the hideTableHeader prop to handle the table header hide and show.
